### DISCUSSION TITLE
Error in GDAS forcing resolution

### DISCUSSION THREAD

**Discussion by MariRiba** [NASA-LIS/LISF#1439] [Q&A] [answered]
Hi 
When i am running LIS, I am getting the following error

[INFO] Using GDAS forcing
 [INFO] GDAS forcing directory : /lustre/mari/gdas
 MSG: The GDAS forcing resolution is coarser than the running domain.
      Interpolating with the neighbor method.
 [INFO] Reading Noah-3.6 vegetation parameter file: ./INPUT/LS_PARAMETERS/noah36_parms/VEGPARM.TBL
 [INFO] Noah-3.6 Landuse type MODI - found           20  categories
 [INFO] Reading Noah-3.6 greenness fraction from greenness maps
 [INFO] Reading SHDMAX map from ./lis_input.gdas.noah36.d01.nc
 [INFO] Reading SHDMIN map from ./lis_input.gdas.noah36.d01.nc
 [INFO] Reading Noah-3.6 albedo from albedo maps
 [INFO] Calculating Noah-3.6 LAI from greenness and LAI min/max in table
 [INFO] Reading Noah-3.6 max snow albedo from max snow albedo maps
 [INFO] noah36_setsoils -- reading soil files
 [INFO] Fixing Noah-3.6 slope index to type:            1
 [INFO] Reading Noah-3.6 soil parameter file: ./INPUT/LS_PARAMETERS/noah36_parms/SOILPARM.TBL.WRF-3.9
 [INFO] Reading Noah-3.6 general parameter file: ./INPUT/LS_PARAMETERS/noah36_parms/GENPARM.TBL
 [INFO] noah36_coldstart -- cold-starting Noah-3.6
 [INFO] noah36_coldstart -- Using the specified start time    2015.0109589041097     
 [INFO]: LIS timestep    900.000000    
[INFO] LIS cycle time: 01/05/2015 00:15:00
 [INFO] Reading greenness map for month           12 from ./lis_input.gdas.noah36.d01.nc
 [INFO] Reading greenness map for month            1 from ./lis_input.gdas.noah36.d01.nc
 [INFO] Reading albedo map for month           12
 [INFO] Reading albedo map for month            1
 [INFO] GDAS -- changing grid to 2010 - 2015 (T574) --
 MSG: The GDAS forcing resolution is coarser than the running domain.
      Interpolating with the neighbor method.
 [INFO] Getting new time1 data
 [INFO] Reading GDAS file1 (I) /lustre/mari/gdas/201501/2015010500.gdas1.sfluxgrbf00.sg
 [INFO] Reading GDAS file1 (A) /lustre/mari/gdas/201501/2015010418.gdas1.sfluxgrbf03.sg
 [INFO] Reading GDAS file1 (A)/lustre/mari/gdas/201501/2015010418.gdas1.sfluxgrbf06.sg
 [ERR] The specified spatial interpolation option 
 [ERR] is not supported for GDAS.
 [ERR] LIS is stopping.
 [ERR] endrun is being called

lislog.oooo and lis.config_noah36_ol is attached below
[lis.config_noah36_ol.tar.gz](https://github.com/NASA-LIS/LISF/files/13178083/lis.config_noah36_ol.tar.gz)
[lislog.tar.gz](https://github.com/NASA-LIS/LISF/files/13178100/lislog.tar.gz)

I am using 
LIS domain
Run domain resolution (dx):             0.081
Run domain resolution (dy):             0.081

As I need to run for 9 KM resolution
[ldt.config.noah36_params.tar.gz](https://github.com/NASA-LIS/LISF/files/13178161/ldt.config.noah36_params.tar.gz)
[ldtlog.0000.tar.gz](https://github.com/NASA-LIS/LISF/files/13178195/ldtlog.0000.tar.gz)

These are the metforcing interpolation options used 

Number of met forcing sources:          1
Blending method for forcings:           "overlay"
Met forcing sources:                    "GDAS" #"NLDAS2"
Topographic correction method (met forcing):    "none"
Enable spatial downscaling of precipitation:    0    0
Spatial interpolation method (met forcing):     "neighbor"
Spatial upscaling method (met forcing):         "average"
Temporal interpolation method (met forcing):    "trilinear"  (sorry !!!  this is supposed to be "linear" )

update:
It is working fine for 
LIS domain
Run domain resolution (dx):             0.25
Run domain resolution (dy):             0.25

Please help me how to resolve this error
Thank you
Mari Riba

**Comment by dmocko** [comment, marked as answer]
Hi @MariRiba 

The error message lets you know that "neighbor" is not supported for GDAS spatial interpolation.

Looking further at the code (and the error message):
https://github.com/NASA-LIS/LISF/blob/master/lis/metforcing/gdas/read_gdas.F90#L513-L544

You can see that these options are supported: `"bilinear"`, `"budget-bilinear"`, and `"average"`.

Please change to one of these options and try again.

**Reply by MariRiba** [reply]
Hi @dmocko 
Thank you for the solution . I was actually following the LIS User guide and I guess I missed the part where it says "neighbor"  is not supported for GDAS spatial interpolation.
I changed "neighbor" to "bilinear" and it is running perfectly.

Thanks again
Mari Riba

**Reply by dmocko** [reply]
You didn't miss it in the Guide, as it's not in there.  Unfortunately, we can't document every combination of options, and sometimes users just need to search through the code for the error message in the lislog or ldtlog file, and examine the code as to why the error message was given.
